### ISSUE TITLE
bytestring constants don't work

### ISSUE THREAD

**Issue by charles-cooper** [vyperlang/vyper#1856]
### Version Information

* vyper Version (output of `vyper --version`): v0.1.0b16+commit.6ea96f0

### What's your issue about?
using bytes or string constants doesn't work. e.g.

```python
NAME: constant(string[5]) = "hello"

@public
def get_name() -> string[64]:
    xs: string[5] = NAME
    return xs
```

### How can it be fixed?
add cases to `make_byte_array_copier`

**Comment by jacqueswww** [comment]
agreed; this should be fixed.

**Comment by iamdefinitelyahuman** [comment]
Fixed in #2029
